### PR TITLE
docs: ✏️ fleek network node health checks

### PR DESCRIPTION
**Pull request by punkbit**

### CODE REVIEW

File: content/posts/fleek-network-node-healthchecks.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+[Log messages](#log-messages) are well formatted, with an identifier describing the type: Warning, Error, etc.
+
+As `Ursa CLI` is in constant development, at the current development stage the output from the Node should be super verbose.
+
+This is to help the development team get feedback. You might see `logs` of the types: Debug, Trace, etc; which for a non-developer human, can cause the feeling of reading the most dreadful poetry in literature, it'd only spark joy to help troubleshoot or make development decisions. As in any book title and book content, feel free to ignore it but don't judge the book by its cover!
+
+### Processes
+
+We recommend running the Stack (for docker-compose users), which provides a proxy, HTTPS, monitoring and analytics capability to your server that is running the Node. You can find instructions on how to run the `Stack` [here](#fleek-network-running-a-node-in-a-docker-container)!
```
Comment (matthias-wright) [inline content/posts/fleek-network-node-healthchecks.md:74]: The link to the `Stack` isn't working in preview mode. Maybe it works once the blog post is published?

File: content/posts/fleek-network-node-healthchecks.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+A Fleek Network Node, or the process we refer to as Node has bound to `0.0.0.0` and has a few ports exposed to the host, these are the port 4069, 6009 and 8070.
+
+Below, we explain what these are used for:
+
+- Port `4069` (TCP), used for HTTP RPC, RPC, REST and metrics
+- Port `6009` (TCP/UDP), used by the P2P protocol running in the network
+- Port `8070` (TCP), used by the HTTP-based index provider
+
+💡 To communicate, the Node uses TCP and UDP (retransmission of lost data packets is only possible with TCP, for example, when we download a file from the internet through our browsers we expect a complete file, no bits should be missing, TCP ensures that the data is received correctly, data is not missing and is in order).
+
```
Comment (matthias-wright) [inline content/posts/fleek-network-node-healthchecks.md:140]: Maybe we can mention that ursa also supports QUIC (UDP port 4890 by default)?

File: content/posts/fleek-network-node-healthchecks.md (hunk 1 above)
Comment (punkbit) [reply content/posts/fleek-network-node-healthchecks.md:74]: Yes, that'll be the case @matthias-wright

File: content/posts/fleek-network-node-healthchecks.md (hunk 2 above)
Comment (punkbit) [reply content/posts/fleek-network-node-healthchecks.md:140]: Was it introduced recently?

I don't have knowledge on that, mind share your thoughts maybe or some source code and I'll write

Comment (punkbit) [reply content/posts/fleek-network-node-healthchecks.md:140]: @matthias-wright I left a message on discord instead.

File: content/posts/fleek-network-node-healthchecks.md [hunk 3]
```diff
@@ -145,47 +153,32 @@ For the ones who followed the [getting started guide](#fleek-network-getting-sta
 We execute a `cURL` request with the `--head` or `-I` flag to show the document info only, in our case the headers of our HTTP response.
 
 ```sh
-curl --head 127.0.0.1:4069
+curl --head 127.0.0.1:80
```
Comment (ozwaldorf) [inline content/posts/fleek-network-node-healthchecks.md]:
```suggestion
curl 127.0.0.1/ping
```

80 is assumed by curl and the ping endpoint will respond with `pong` and code 200

File: content/posts/fleek-network-node-healthchecks.md [hunk 4]
```diff
@@ -145,47 +153,32 @@ For the ones who followed the [getting started guide](#fleek-network-getting-sta
...
 ```
 
 Or, the shorter version:
 
 ```sh
-curl -I 127.0.0.1:4069
+curl -I 127.0.0.1:80
```
Comment (ozwaldorf) [inline content/posts/fleek-network-node-healthchecks.md]:
```suggestion
```

can remove

File: content/posts/fleek-network-node-healthchecks.md [hunk 5]
```diff
@@ -145,47 +153,32 @@ For the ones who followed the [getting started guide](#fleek-network-getting-sta
...
 
 ```sh
-HTTP/1.1 405 Method Not Allowed
-content-length: 0
-date: Wed, 04 Jan 2023 19:00:52 GMT
+HTTP/1.1 404 Not Found
+Server: nginx/1.23.3
+Date: Fri, 06 Jan 2023 18:36:09 GMT
+Content-Length: 0
+Connection: keep-alive
```
Comment (ozwaldorf) [inline content/posts/fleek-network-node-healthchecks.md]: if setting to just show the /ping response

```suggestion
pong
```

OR

if we want to keep this checkup to headers only, this is what /ping would result wiht

```suggestion
HTTP/1.1 200
Server: nginx/1.23.3
Date: Fri, 06 Jan 2023 18:36:09 GMT
Content-Length: 4
Connection: keep-alive
```

Comment (ozwaldorf) [inline content/posts/fleek-network-node-healthchecks.md:174]:
```suggestion
HTTP/1.1 200
Server: nginx/1.23.3
Date: Fri, 06 Jan 2023 18:36:09 GMT
Content-Length: 4
Connection: keep-alive
```

or if we want to keep this checkup to headers only, this is what /ping would result wiht

File: content/posts/fleek-network-node-healthchecks.md [hunk 6]
```diff
@@ -145,47 +153,32 @@ For the ones who followed the [getting started guide](#fleek-network-getting-sta
...
-date: Wed, 04 Jan 2023 19:09:23 GMT
-```
-
-You can determine failure when you make a `cURL` request which fails:
-
-```sh
-curl: (7) Failed to connect to 127.0.0.1 port 4069: Connection refused
-curl: (7) Failed to connect to 127.0.0.1 port 6009: Connection refused
-curl: (7) Failed to connect to 127.0.0.1 port 8070: Connection refused
-```
```
Comment (ozwaldorf) [inline content/posts/fleek-network-node-healthchecks.md]: revert this is relevant still

````suggestion
You can determine failure when you make a `cURL` request which fails:

```sh
curl: (7) Failed to connect to 127.0.0.1 port 80: Connection refused
curl: (7) Failed to connect to 127.0.0.1 port 6009: Connection refused
```
````

**Review by ozwaldorf — APPROVED** [review]

**Pull request merged**